content/en/serverless/azure_container_apps/_index.md

```diff
@@ -143,3 +147,6 @@ You can use the [Azure integration][1] to collect logs. Alternatively, you can s
 [4]: /metrics/custom_metrics/dogstatsd_metrics_submission/
 [5]: /getting_started/site/
 [6]: /getting_started/tagging/unified_service_tagging/
+[7]: /serverless/custom_metrics/#understanding-distribution-metrics
```

Comment:
The rest of the content of that page is about Lambda, so I'm worried that linking user from a azure container apps doc to a lambda doc could cause confusion. I think we can probably leave out the link or we should refactor that section from the lambda doc to a standalone guide.